nucleus/validate/eval_functions/available_eval_functions.py

```diff
@@ -11,30 +11,147 @@
 
 
 class BoundingBoxIOU(BaseEvalFunction):
+    def __call__(
+        self,
+        enforce_label_match: bool = False,
+        iou_threshold: float = 0.0,
+        confidence_threshold: float = 0.0,
+        **kwargs,
+    ):
+        """Configures a call to :class:`PolygonIOU` object.
+
+        Args:
+            enforce_label_match: whether to enforce that annotation and prediction labels must match. Defaults to False
+            iou_threshold: IOU threshold to consider detection as valid. Must be in [0, 1]. Default 0.0
+            confidence_threshold: minimum confidence threshold for predictions. Must be in [0, 1]. Default 0.0
+        """
+        return super().__call__(
+            enforce_label_match=enforce_label_match,
+            iou_threshold=iou_threshold,
+            confidence_threshold=confidence_threshold,
+            **kwargs,
+        )
+
     @classmethod
     def expected_name(cls) -> str:
         return "bbox_iou"
 
 
 class BoundingBoxMeanAveragePrecision(BaseEvalFunction):
+    def __call__(
+        self,
+        iou_threshold: float = 0.0,
+        **kwargs,
+    ):
+        """Configures a call to :class:`PolygonMAP` object.
+
+        Args:
+            iou_threshold: IOU threshold to consider detection as valid. Must be in [0, 1]. Default 0.0
+        """
+        return super().__call__(
+            iou_threshold=iou_threshold,
+            **kwargs,
+        )
+
     @classmethod
     def expected_name(cls) -> str:
         return "bbox_map"
 
 
 class BoundingBoxRecall(BaseEvalFunction):
+    def __call__(
+        self,
+        enforce_label_match: bool = False,
+        iou_threshold: float = 0.0,
```

Comment:
Isn't the threshold for detection matching under the hood 0.5? Then we should go for default 0.5 here too, otherwise this doesn't make sense

Reply:
Not sure, the default parameters are copied from the `nucleus/metrics` module. I'll check!

Reply:
Yeah, copy paste error on my side. Good catch 🙂